src/Redux.elm

```diff
@@ -24,25 +25,20 @@ view model =
 program :
     { init : ( model, Cmd msg )
     , update : msg -> model -> ( model, Cmd msg )
+    , encode : model -> Json.Value
```

Comment:
I mean in redux.js this json-stri g should be decoded/parsed again (before dispatching @elm/ACTION), so that the js world gets the new model as a object and not as a encoded json.